Source/ACE/Managers/LandblockManager.cs

```diff
@@ -52,6 +52,17 @@ public static void RelocateObject(WorldObject worldObject)
         }
 
         /// <summary>
+        /// Find a worldobject by its guid in the specified landblock
+        /// </summary>
+        public static WorldObject GetObjectByGuid(LandblockId blockId, ObjectGuid objectId)

```

Comment:
Remove this method.  This might be needed eventually, but for now it's a red flag for "you're doing it wrong."